src/common/Page.ts

```diff
@@ -1795,12 +1795,12 @@ export class Page extends EventEmitter {
    *   more than 2 network connections for at least `500` ms.
    */
   async reload(options?: WaitForOptions): Promise<HTTPResponse | null> {
-    const result = await Promise.all<HTTPResponse, void>([
+    const result = await Promise.all<HTTPResponse | void>([
```

Comment:
was it intended to be `Promise.all<[HTTPResponse, void]>` instead? (if that compiles)

Reply:
This will no longer compile in new versions of typescript.